examples/ephemeral_gpu_offloading/load_with_dora.py

```diff
@@ -0,0 +1,90 @@
+# Example script demonstrating the time difference loading a model with a DoRA using ephemeral GPU offloading
+# vs doing it purely on the CPU.
+
+import argparse
+import time
+
+from huggingface_hub import snapshot_download
+from transformers import AutoModelForCausalLM
+
+from peft import PeftModel
+
+
+def main():
+    parser = argparse.ArgumentParser(description="Load a model with DoRA using ephemeral GPU offloading")
+    parser.add_argument("--model", type=str, default="NousResearch/Hermes-2-Pro-Mistral-7B", help="Model to load")
+    parser.add_argument(
+        "--dora",
+        type=str,
+        default="anamikac2708/Mistral-7B-DORA-finetuned-investopedia-Lora-Adapters",
```

Comment:
I'd feel a bit more comfortable pointing this to a PEFT-internal repo. I created a DoRA adapter for this with the same size:

`peft-internal-testing/DoRA-Hermes-2-Pro-Mistral-7B`

This adapter is not trained but for the purpose of this example, I don't think it matters.

Reply:
Thank you! I didn't think of just doing a non-trained DoRA! Couldn't find one readily available.